### PR TITLE
Feature/test editor

### PR DESCRIPTION
**Pull request by ayushaga14**

### CODE REVIEW

File: libs/dao/src/main/java/com/akto/dto/test_editor/sampleYaml.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+id: jwt-none-algo
```
Comment (ankush-jain-akto) [inline libs/dao/src/main/java/com/akto/dto/test_editor/sampleYaml.yaml:1]: We need this file? If no, can we remove it please ?

File: apps/testing/src/test/java/com/akto/rules/TestFilterTest.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
+package com.akto.rules;
```
Comment (ankush-jain-akto) [inline apps/testing/src/test/java/com/akto/rules/TestFilterTest.java:1]: remove this file if it doesn't do anything?

File: apps/testing/src/main/java/com/akto/testing_issues/TestingIssuesHandler.java [hunk 3]
```diff
@@ -106,16 +109,18 @@ private void insertVulnerableTestsIntoIssuesCollection(List<WriteModel<TestingRu
...
                     TestSourceConfig config = TestSourceConfigsDao.instance.getTestSourceConfig(runResult.getTestSubType());
                     writeModelList.add(new InsertOneModel<>(new TestingRunIssues(testingIssuesId,
                             config.getSeverity(),
                             TestRunIssueStatus.OPEN, lastSeen, lastSeen, runResult.getTestRunResultSummaryId())));
                 }else {
                     writeModelList.add(new InsertOneModel<>(new TestingRunIssues(testingIssuesId,
-                            subCategory.getSuperCategory().getSeverity(),
-                            TestRunIssueStatus.OPEN, lastSeen, lastSeen, runResult.getTestRunResultSummaryId())));
+                            GlobalEnums.Severity.HIGH,
+                            TestRunIssueStatus.OPEN, lastSeen, lastSeen, runResult.getTestRunResultSummaryId()))); // todo: take value from yaml
```
Comment (ankush-jain-akto) [inline apps/testing/src/main/java/com/akto/testing_issues/TestingIssuesHandler.java:123]: pending todo here?

File: apps/testing/src/main/java/com/akto/test_editor/filter/data_operands_impl/GreaterThanFilter.java [hunk 4]
```diff
@@ -0,0 +1,40 @@
+package com.akto.test_editor.filter.data_operands_impl;
+
```
Comment (ankush-jain-akto) [inline apps/testing/src/main/java/com/akto/test_editor/filter/data_operands_impl/GreaterThanFilter.java:2]: refactor this please ?

File: apps/testing/src/main/java/com/akto/testing_issues/TestingIssuesHandler.java (hunk 3 above)
Comment (ayushaga14) [reply apps/testing/src/main/java/com/akto/testing_issues/TestingIssuesHandler.java:123]: can be ignored for now

File: apps/testing/src/main/java/com/akto/test_editor/filter/data_operands_impl/GreaterThanFilter.java (hunk 4 above)
Comment (ayushaga14) [reply apps/testing/src/main/java/com/akto/test_editor/filter/data_operands_impl/GreaterThanFilter.java:2]: done

**Review by ankush-jain-akto — APPROVED** [review]

**Review by ankush-jain-akto — APPROVED** [review]

**Pull request merged**